src/styles/page.scss

```
@@ -0,0 +1,91 @@
+.home-page {
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  --main-color: #060b35;
+  --second-color: #616070;
```

```suggestion
  $main-color: #060b35;
  $second-color: #616070;
```
I would suggest using separate file for variables and using scss syntax instead css